src/main/java/uk/gov/hmcts/reform/professionalapi/domain/BulkCustomerDetails.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.professionalapi.domain;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.Size;
+
+import static javax.persistence.GenerationType.AUTO;
+
+@Entity(name = "bulk_customer_details")
+@NoArgsConstructor
+@Getter
+@Setter
+@EntityListeners(AuditingEntityListener.class)
+public class BulkCustomerDetails implements Serializable {
```

Comment:
@manukundloo-hmcts Can you please move this class inside dataload folder

Reply:
hi @arshinsalim no dataload folder in prd api all entity classes in domain folder only.